src/app/forms/custom-sections/letters/letters.component.html

```diff
@@ -0,0 +1,24 @@
+<form [formGroup]="form">
+  <dl class="govuk-summary-list">
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Letter type</dt>
+      <dd id="test-cor" class="govuk-summary-list__value">{{ letterType ?? '-' }}</dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Date requested</dt>
+      <dd id="test-cor" class="govuk-summary-list__value">{{ dateRequested ?? '-' }}</dd>
+    </div>
+    <div class="govuk-summary-list__row">
+      <dt class="govuk-summary-list__key">Letter:</dt>
+      <dd id="test-certificate" class="govuk-summary-list__value">
+        <button id="test-certificate-link" class="link" (click)="download()">TEST LETTER</button>
+      </dd>
+    </div>
+    <div class="govuk-grid-row">
+      <br />
+      <div class="govuk-grid-column-full">
+        <app-button (clicked)="generateLetter()" *appRoleRequired="roles.TechRecordAmend">Generate letter</app-button>
```

Comment:
```suggestion
        <app-button *appRoleRequired="roles.TechRecordAmend" routerLink="generate-letter">Generate letter</app-button>
```
This way you can omit the method in the component.